src/lib/realmdb/RealmDB.js

```diff
@@ -93,13 +96,31 @@ class RealmDB implements DB {
     }
   }
 
+  /**
+   * helper to resolve issue with toJSON error in console
+   * @returns {Realm.Services.MongoDB.MongoDBCollection<any>}
+   * @private
+   */
+  _encryptedFeed() {
```

Comment:
Maybe it's possible to make them get() props. Technically this syntax sugar it referring to the get function call each time you're accessing the property